docs/precommit.md

```diff
@@ -11,11 +11,36 @@ You can use Prettier with a pre-commit tool. This can re-format your files that
 
 _Make sure Prettier is installed and is in your [`devDependencies`](https://docs.npmjs.com/specifying-dependencies-and-devdependencies-in-a-package-json-file) before you proceed._
 
+Install [husky](https://github.com/typicode/husky) and [lint-staged](https://github.com/okonet/lint-staged):
+
+<!--DOCUSAURUS_CODE_TABS-->
+<!--npm-->
+
 ```bash
-npx mrm@2 lint-staged
+npx husky-init
+npm install --save-dev lint-staged
+npx husky set .husky/pre-commit "npx lint-staged"
 ```
 
-This will install [husky](https://github.com/typicode/husky) and [lint-staged](https://github.com/okonet/lint-staged), then add a configuration to the project’s `package.json` that will automatically format supported files in a pre-commit hook.
+<!--yarn-->
+
+```bash
+npx husky-init # add --yarn2 for Yarn 2
+yarn add --dev lint-staged
+yarn husky set .husky/pre-commit "npx lint-staged"
+```
+
+<!--pnpm-->
+
+```bash
+pnpm exec husky-init
```

Comment:
```suggestion
pnpm dlx husky-init && pnpm install 
```

https://typicode.github.io/husky/#/?id=automatic-recommended

Reply:
If case this is correct or better, it should be applied to the other Options that use husky-init

Reply:
The problem is in the setup step where we don't specify to install `husky-init`. Therefore, when you run `pnpm exec husky-init`, it fails, which is why dlx is required.

> [pnpm exec](https://pnpm.io/cli/exec)
> Execute a shell command in scope of a project.

> [pnpm dlx](https://pnpm.io/cli/dlx)
> Fetches a package from the registry without installing it as a dependency, hotloads it, and runs whatever default command binary it exposes.